lib/gui/app.js

```diff
@@ -36,6 +36,7 @@ const app = angular.module('Etcher', [
   require('angular-ui-router'),
   require('angular-ui-bootstrap'),
   require('angular-if-state'),
+  require('angular-sanitize'),
```

Comment:
Bringing in `angular-sanitize` is probably overkill for this purpose (e.g: we're not dealing with complex whitelists, `$sanitize`, etc, which is what this module shines for). You can make this work using what Angular already provides by calling `$sce.trustAsHtml` on the HTML string.

I guess you can do something like `options.description = $sce.trustAsHtml(options.description)` on `WarningModalService`, or you could expose `this.template = $sce.trustAsHtml(options.description)` on the controller and use like `<p ng-bind-html="modal.template"></p>`.